log4j-spring-cloud-config-client/pom.xml

```diff
@@ -120,33 +86,21 @@
     </dependency>
 
     <dependency>
-      <groupId>org.junit.vintage</groupId>
-      <artifactId>junit-vintage-engine</artifactId>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-jcl</artifactId>
```

Comment:
When working on #1876, I noticed that the tests in `log4j-jcl` gave a false positive, because `spring-jcl` was covering for it, so I banned all external logging backends/bridges:

https://github.com/apache/logging-log4j2/blob/252f1a65e1b44fa78c622b4140d9116bf57d3617/log4j-parent/pom.xml#L1172-L1188

However Spring does actually need an implementation of Commons Logging on the classpath. `log4j-jcl` seemed a more elegant choice than `commons-logging:commons-logging` ;-)

Reply:
Done